src/screens/Gps.tsx

```diff
@@ -192,6 +193,14 @@ export default class Gps extends Component<IGpsProps, IGpsState> {
   render() {
     return (
       <View style={GpsStyles(this.context).screen.style.container}>
+        <CWWidget
+          style={GpsStyles(this.context).weatherWidget}
+          data={{
+            city: 'Colima',
+            degrees: '80',
+            icon: 'https://www.losmundosdenoa.es/wp-content/uploads/2020/06/sol-e1603880900911.png',
+          }}
```

Comment:
What do you think about importing this information from a style file?
```suggestion
          data={style.data}
```

Reply:
> What do you think about importing this information from a style file?

This is going to come from the state, it's not already set, but it will be replaced. It's just an example for the moment. 😶‍🌫️